tests/bin/test_tmux_layout_integration.py

```
@@ -1,289 +1,289 @@
+            
+            # ペイン数を確認
+            cmd = f"tmux list-panes -t {test_session}:MAIN -F '#{{pane_index}}:#{{pane_width}}x#{{pane_height}}'"
+            result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
+            
+            if result.returncode == 0:
+                panes = result.stdout.strip().split('\n')
+                print(f"  Total panes: {len(panes)}")
+                for pane in panes:
+                    print(f"    {pane}")
+                
+                # 期待値：初期3ペイン + 子3つで分割 = 5ペイン以上
```

Ruff (0.12.2)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)